### PR TITLE
No more 4.1 runs on next, just ocp 4.2

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -45,6 +45,5 @@ generate-release:
 
 # Generates a ci-operator configuration for a specific branch.
 generate-ci-config:
-	./openshift/ci-operator/generate-ci-config.sh $(BRANCH) 4.1 > ci-operator-config_41.yaml
-	./openshift/ci-operator/generate-ci-config.sh $(BRANCH) 4.2 > ci-operator-config_42.yaml
+    ./openshift/ci-operator/generate-ci-config.sh $(BRANCH) > ci-operator-config.yaml
```
Comment (syedriko) [inline Makefile]: Assuming the latest version is the default?

Comment (matzew) [reply Makefile]: thats invoked via tools, and passed in

**Pull request merged**